plenum/server/replica.py

```diff
@@ -904,7 +917,7 @@ def __is_next_pre_prepare(self, view_no: int, pp_seq_no: int):
             logger.debug('{} missing PRE-PREPAREs between {} and {}'.
                          format(self, pp_seq_no, last_pp_seq_no))
             # TODO: think of a better way, urgently
-            self.set_last_ordered_for_non_master()
+            self._set_last_ordered_for_non_master()
```

Comment:
Agree, but there is strong expectation that a function with `is_*` name only returns a bool value with out side affect. I'd be too surprise if such function changes the world.